tools/pytorch2onnx.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+
+import mmcv
+import numpy as np
+import onnx
+import onnxruntime as rt
+import torch
+from mmcv.runner import load_checkpoint
+from mmedit.datasets.pipelines import Compose
+from mmedit.models import build_model
+from onnx_util.symbolic import register_extra_symbolics
+
+
+def pytorch2onnx(model,
+                 input,
+                 opset_version=11,
+                 show=False,
+                 output_file='tmp.onnx',
+                 verify=False):
+    model.cpu().eval()
+    merged = input['merged'].unsqueeze(0)
+    trimap = input['trimap'].unsqueeze(0)
+    input = torch.cat((merged, trimap), 1)
+    model.forward = model.forward_dummy
+    # pytorch has some bug in pytorch1.3, we have to fix it
+    # by replacing these existing op
+    register_extra_symbolics(opset_version)
+    with torch.no_grad():
+        torch.onnx.export(
+            model,
+            input,
+            output_file,
+            input_names=['cat_input'],
+            export_params=True,
+            keep_initializers_as_inputs=True,
+            verbose=show,
+            opset_version=opset_version)
+    print(f'Successfully exported ONNX model: {output_file}')
+    if verify:
+        # check by onnx
+        onnx_model = onnx.load(output_file)
+        onnx.checker.check_model(onnx_model)
+
+        # get pytorch output, only concern pred_alpha
+        pytorch_result = model(input)
+        if isinstance(pytorch_result, (tuple, list)):
+            pytorch_result = pytorch_result[0]
+        pytorch_result = pytorch_result.detach().numpy()
+        # get onnx output
+        sess = rt.InferenceSession(output_file)
+        onnx_result = sess.run(None, {
+            'cat_input': input.detach().numpy(),
+        })
+        # only concern pred_alpha value
+        if isinstance(onnx_result, (tuple, list)):
+            onnx_result = onnx_result[0]
+        # check the numerical value
+        assert np.allclose(
+            pytorch_result,
+            onnx_result), 'The outputs are different between Pytorch and ONNX'
+        print('The numerical values are same between Pytorch and ONNX')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Convert MMDet to ONNX')
```

Comment:
Rename `MMDet` to `MMEdit` accordingly?

Reply:
> Rename `MMDet` to `MMEditing` accordingly?

Ok, I will fix it.
BTW, as I have just decided to move symblic.py and other file to mmcv, so I will submit a new commit after my corresponding PR merged to MMCV